### PR TITLE
Added restaurants info

### PR DESCRIPTION
**Pull request by skdharma**
Please provide feedback on:
- Which restaurant do you like?
- What else you want me to add?

### CODE REVIEW

File: skdharma-fremont.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Fremont, CA
+
+### Restaurants
+
+- Indian
+- Veg
+- Non-veg
+
```
Comment (diarmaidm) [inline skdharma-fremont.md]: Unfortunately I don't eat any veg. Any meat? :)

**Comment by diarmaidm** [comment]
> Please provide feedback on:
> 
> Which restaurant do you like?
> What else you want me to add?

This looks good

**Comment by skdharma** [comment]
@diarmaidm added your suggestions

**Comment by fluyapstone** [comment]
👍

**Pull request merged**